README.md

```diff
@@ -27,7 +27,45 @@ add the code snippet from https://github.com/SWAuk/plg_swa_viewlevels
 The original template sources is https://www.favthemes.com/joomla-templates/product/favourite-free-responsive-joomla-3-template
 A copy of this can be found on the SWA Drive "Tech/Web Dev/Templates/favourite_j!3_UnzipFirst.zip"
 
-## Development Environment
+## Development - Dependencies
+
+### composer
+
+In order to install the needed packages for development you'll need [composer](https://getcomposer.org/download/).
+Composer is a dependency manager and command runner for PHP.
+
+Use it to install the PHP libraries needed for testing, development and production.
+
+As docker is already required, the easiest way to use `composer` to install the required libs is via the official [docker image](https://hub.docker.com/_/composer).
+
+You can also run **composer in docker**...
```

Comment:
This line now not needed as the line above explains it better?

Reply:
Will do